sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -40,62 +40,59 @@ private RequestValidator() {}
       for (Map<String, Object> map : list) {
         if (null != map.get(JsonKey.LAST_UPDATED_TIME)) {
           boolean bool =
-              ProjectUtil.isDateValidFormat(
-                  "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_UPDATED_TIME));
+                  ProjectUtil.isDateValidFormat(
+                          "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_UPDATED_TIME));
           if (!bool) {
             throw new ProjectCommonException(
-                ResponseCode.dateFormatError.getErrorCode(),
-                ResponseCode.dateFormatError.getErrorMessage(),
-                ERROR_CODE);
+                    ResponseCode.dateFormatError.getErrorCode(),
+                    ResponseCode.dateFormatError.getErrorMessage(),
+                    ERROR_CODE);
           }
         }
         if (null != map.get(JsonKey.LAST_COMPLETED_TIME)) {
           boolean bool =
-              ProjectUtil.isDateValidFormat(
-                  "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_COMPLETED_TIME));
+                  ProjectUtil.isDateValidFormat(
+                          "yyyy-MM-dd HH:mm:ss:SSSZ", (String) map.get(JsonKey.LAST_COMPLETED_TIME));
           if (!bool) {
             throw new ProjectCommonException(
-                ResponseCode.dateFormatError.getErrorCode(),
-                ResponseCode.dateFormatError.getErrorMessage(),
-                ERROR_CODE);
+                    ResponseCode.dateFormatError.getErrorCode(),
+                    ResponseCode.dateFormatError.getErrorMessage(),
+                    ERROR_CODE);
           }
         }
         if (map.containsKey(JsonKey.CONTENT_ID)) {
 
           if (null == map.get(JsonKey.CONTENT_ID)) {
             throw new ProjectCommonException(
-                ResponseCode.contentIdRequired.getErrorCode(),
-                ResponseCode.contentIdRequiredError.getErrorMessage(),
-                ERROR_CODE);
+                    ResponseCode.contentIdRequired.getErrorCode(),
+                    ResponseCode.contentIdRequiredError.getErrorMessage(),
+                    ERROR_CODE);
           }
           if (ProjectUtil.isNull(map.get(JsonKey.STATUS))) {
             throw new ProjectCommonException(
-                ResponseCode.contentStatusRequired.getErrorCode(),
-                ResponseCode.contentStatusRequired.getErrorMessage(),
-                ERROR_CODE);
+                    ResponseCode.contentStatusRequired.getErrorCode(),
+                    ResponseCode.contentStatusRequired.getErrorMessage(),
+                    ERROR_CODE);
           }
 
         } else {
           throw new ProjectCommonException(
-              ResponseCode.contentIdRequired.getErrorCode(),
-              ResponseCode.contentIdRequiredError.getErrorMessage(),
-              ERROR_CODE);
+                  ResponseCode.contentIdRequired.getErrorCode(),
```

Comment:
contentIdRequired exception can be common for above code

Reply:
This was existing code. We didn't change anything.